Orange/data/tests/test_table.py

```diff
@@ -380,8 +380,10 @@ def test_unpickled_owns_data(self):
             Table.LOCKING = default
 
         unpickled = pickle.loads(pickle.dumps(table))
-        self.assertFalse(unpickled.is_view())
-        self.assertTrue(unpickled.is_copy())
+        self.assertFalse(unpickled.is_sparse())
+        self.assertTrue(unpickled.X.shape[-1] or unpickled.X.base is None)
```

Comment:
What is the reason that you are testing additional properties for X? If this is indeed needed, please make a comment in the code that explains the reasoning.

Reply:
Just to stay consistent with what is_view and is_copy returned. I assumed there was a reason I just couldn't think of one.
If there is no reason and all we need is to check if `unpickled` actually owns it's data then as far as I can tell it would be enough to only check all `base` attributes.

Reply:
Yes, you should probably only do the latter...